src/main/resources/view/DarkTheme.css

```diff
@@ -137,7 +137,8 @@
 }
 /*@@author*/
 
-.list-cell:filled #insuranceCardPane .particular-link:hover {
+/*@@author RSJunior37*/
+.list-cell:filled #insuranceCardPane .particular-person:hover {
```

Comment:
Apologies for the mistake made when resolving conflicts. I will double check and correct after merging this PR.